test/fixtures/user/user.js

```diff
@@ -68,6 +68,7 @@ module.exports = () => {
       github_display_name: "Sagar Bajpai",
       phone: "1234567890",
       email: "abc@gmail.com",
+      status: "active",
```

Comment:
Could you please explain the reason for which this was changed?

Reply:
because we want to test if the user has active status then we do not want to give them a task that is why I changed it